Add "Next steps" page to "An introduction to Daml" (#14899)
digital-asset/daml: docs/source/daml/intro/99_NextSteps.rst

.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
.. SPDX-License-Identifier: Apache-2.0

Next Steps
==========

Now that you have through the entirety of this introduction to the Daml smart contract language,
you might be interested in what to do next. It depends on what you would like to do with Daml:

- What you have learned so far should be enough to enable to you become a certified Daml modeler.
  You can test your skills on the page about `Daml certifications <https://www.digitalasset.com/developers/certifications>`_.

- If you want to improve your understanding of proven design patterns, you can learn more at

>   :doc:`this </daml/patterns>` page.

Reviewer: ```suggestion
  :doc:`the Patterns </daml/patterns>` page.
```

- If you're interested in building off-ledger services that interact and integrate with your

>   on-ledger Daml models, you read the :doc:`"Building Applications" section </app-dev/app-arch>`.

Reviewer: ```suggestion
  on-ledger Daml models, read the :doc:`"Building Applications" section </app-dev/app-arch>`.
```

- If you're interested in understanding how to install, operate and maintain a production-grade Daml
  ledger, you can have a look at the `Canton user manual <../../canton/usermanual/usermanual.html>`_.

> - If you want to build Daml applications in a fully managed environment that relieves you from

Reviewer: ```suggestion
- If you want to build Daml applications in a fully-managed environment that handles
```

>   the day-to-day operation of your Daml ledger, you can start building your Daml application
>   right away on `Daml Hub <https://hub.daml.com>`_.

Reviewer: ```suggestion
  the day-to-day operation of your Daml ledger for you, you can start 
 right away on `Daml Hub <https://hub.daml.com>`_.
```

- If you want want to see more examples of Daml applications to understand what is possible with
  Daml, we have a `a library full of examples <https://www.digitalasset.com/developers/examples>`_

>   for you to study and learn from.

Reviewer: ```suggestion
  for you to study.
```
